### PR TITLE
Add updated recipe for inwx-api-bundle v0.2

### PR DESCRIPTION
**Pull request by SebTM**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: sebtm/inwx-api-bundle/0.2/config/packages/inwx_api.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read the documentation: https://github.com/SebTM/inwx-api-bundle
+inwx_api:
+    #environment: "test"
+    username: "username"
```
Comment (Nyholm) [inline sebtm/inwx-api-bundle/0.2/config/packages/inwx_api.yaml]: You should use environment variables for username/password, right?

**Comment by SebTM** [comment]
@Nyholm Should be fixed now - thx for you hint :)

**Comment by Nyholm** [comment]
Thank you

**Comment by SebTM** [comment]
Thanks for the merge :) 👍

**Pull request merged**